src/static/js/modules/loan-comparison/mortgage-calculations.js

```diff
@@ -80,6 +84,10 @@ mortgage['monthly-principal-interest'] = function (loan) {
 };
 
 mortgage['monthly-mortgage-insurance'] = function (loan) {
+    var monthly = loan['mtg-ins-data'].monthly;
```

Comment:
I wonder if we should ensure we have an object here, just in case:
`(loan['mtg-ins-data'] || {}).monthly`